dls-controls/malcolmjs
# Feature/layout palette #233

## Description

Added a palette to allow blocks to be added and removed.
Major refactor of the layout state so the engine is stored in redux now, this makes event interactions a lot simpler.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run the site and navigate to a layout

## Agile board tracking

connect to #233 
closes #233
